pkgs/applications/graphics/hdrmerge/default.nix

```
@@ -0,0 +1,77 @@
+{ lib
+, mkDerivation
+, fetchpatch
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, qtbase
+, wrapQtAppsHook
+, libraw
+, exiv2
+, zlib
+, alglib
+, pkgconfig
+, makeDesktopItem
+}:
+
+mkDerivation rec {
+  pname = "hdrmerge";
+  version = "unstable-2020-11-12";
+  src = fetchFromGitHub {
+    owner = "jcelaya";
+    repo = "hdrmerge";
+    rev = "f5a2538cffe3e27bd9bea5d6a199fa211d05e6da";
+    sha256 = "1bzf9wawbdvdbv57hnrmh0gpjfi5hamgf2nwh2yzd4sh1ssfa8jz";
+  };
+
+  # dev dependencies
+  nativeBuildInputs = [
+    cmake
+    pkgconfig
+    wrapQtAppsHook
+  ];
+
+  # runtime dependencies
+  buildInputs = [ qtbase libraw exiv2 zlib alglib ];
+
+  cmakeFlags = [
+    "-DALGLIB_DIR:PATH=${alglib}"
+  ];
+
+  patches = [
+    (fetchpatch {
+      # patch FindAlglib.cmake to respect ALGLIB_DIR
+      # see https://github.com/jcelaya/hdrmerge/pull/213
+      name = "patch-hdrmerge-CMake.patch";
+      url = "https://github.com/mkroehnert/hdrmerge/commit/472b2dfe7d54856158aea3d5412a02d0bab1da4c.patch";
+      sha256 = "0jc713ajr4w08pfbi6bva442prj878nxp1fpl9112i3xj34x9sdi";
+    })
+  ];
+
+  desktopItem = makeDesktopItem {
+    name = "HDRMerge";
+    genericName = "HDR raw image merge";
+    desktopName = "HDRMerge";
+    comment = meta.description;
+    icon = "hdrmerge";
+    exec = "@out@/bin/hdrmerge -F";
+    categories = "Graphics;";
+    mimeType = "image/x-dcraw;image/x-adobe-dng";
+    terminal = false;
+  };
+
+  postInstallPhase = ''
+      # Make a desktop item
+      mkdir -p $out/share/icons/ $out/share/applications/
+    cp ../data/images/logo.png $out/share/icons/hdrmerge.png
+      cp ${desktopItem}/share/applications/* $out/share/applications/
+    '';
+
+  meta = with lib; {
+    homepage = https://github.com/jcelaya/hdrmerge;
```

Not sure if this makes a difference, but still.. convention:
```suggestion
    homepage = "https://github.com/jcelaya/hdrmerge";
```

---

Soo, I dug this up: https://github.com/NixOS/rfcs/pull/45